config/detekt/detekt.yml

```diff
@@ -150,7 +150,7 @@ complexity:
     thresholdInObjects: 11
     thresholdInEnums: 11
     ignoreDeprecated: false
-    ignorePrivate: false
+    ignorePrivate: true
```

Comment:
What’s the reason for the changes in this file?

Reply:
Detekt was failing for one of the files for having more than 11 methods, I think ignoring private methods is a good change to make it a little more flexible.